src/pages/home/report/ReactionList/PopoverReactionList.js

```diff
@@ -2,15 +2,58 @@ import React from 'react';
 import {Dimensions} from 'react-native';
 
 import lodashGet from 'lodash/get';
+import lodashMap from 'lodash/map';
+import lodashFilter from 'lodash/filter';
+import lodashFind from 'lodash/find';
+import lodashEach from 'lodash/each';
+import lodashIsEqual from 'lodash/isEqual';
```

Comment:
Using underscore is our standard to keep everything consistent: https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#object--array-methods

Please remove these and use underscore instead.

Reply:
`lodashGet` is the only one we allow since underscore doesn't have an equivalent.

Reply:
@tgolen @rushatgabhane I just created PR to fix: https://github.com/Expensify/App/pull/19183